samples/multiplatform/settings.gradle

```diff
@@ -1,4 +1,9 @@
+enableFeaturePreview('GRADLE_METADATA')
+
 include ':greeting'
 include ':greeting:common'
 include ':greeting:android'
 include ':greeting:ios'
+
+includeBuild '../../shared'
```

Comment:
I would recommend to add a comment here that this is not part "of the sample" and only used for **this** sample...
Know what I mean? 🙃

Reply:
Yes, I see :)

The sample becomes a little bit complicated with all these composite builds. 
May be it's better to move it in a separate repo?

We use composite build to  allow a user to build all samples in the main repo with the compiler and the Gradle plugin built from sources right now. But in a separate repo we can use a fixed KN version and where will be no differences between the documentation and the sample itself. I can make a repo under JB organisation and move your changes into it. What do you think about it?

Reply:
Indeed. The sample blows a little bit 🙃 
Yes, I think it make sense to move it in a new repository. 
The only thing we have to make sure is that the documentation and the new repo are always in sync.
We can also move the documentation into the new repository and link in *this* repo to the new repo?! 🤔 
Having the documentation here but the implementation in a new repo is a little bit strange...

Anyway.
I would welcome to have a separate repository for it.
The rest can be discussed then 👍

Reply:
> Having the documentation here but the implementation in a new repo is a little bit strange
Having all docs in one place is more convenient though. But you're right too and I would rather move the MPP doc in the new repo. @olonho what do you think?